lxc/publish.go

```diff
@@ -33,6 +34,7 @@ func (c *publishCmd) flags() {
 	gnuflag.Var(&c.pAliases, "alias", i18n.G("New alias to define at target"))
 	gnuflag.BoolVar(&c.Force, "force", false, i18n.G("Stop the container if currently running"))
 	gnuflag.BoolVar(&c.Force, "f", false, i18n.G("Stop the container if currently running"))
+	gnuflag.Var(&c.Compression, "compression", i18n.G("Define a compression algorithm: for image or none"))
```

Comment:
You'll need to run "make i18n" and include its result in your commit since you're introducing a new string here.